sisl/io/gulp/got.py

```diff
@@ -255,8 +253,7 @@ def _r_dynamical_matrix_got(self, geom, **kwargs):
 
                         i += 1
                         j = 0
-                        if i >= no:
-                            break
+                        break
```

Comment:
I think this is wrong. This should happen when a new row of *orbitals* occurs in the middle of the 12 entries. Say for 5 atom structures. In that case the `dat` array needs to be filled in the new row.

Reply:
Gulp puts a line break when you have a new row - it doesn't go to the next row in the middle of a 12 entries wide line. So it needs to go the next line. That was actually why it was crashing for me.

Reply:
I was considering putting a test for it (ie just read a .gout file with norbs%12!=0 without crashing), but I don't think I can do it in one PR, right? It has to go to the other repo as well?

Reply:
Yes and no, either send me the test file and I'll do it, or.
1) create a pr with the test file in the other repo
2) amend this pr with the test (and also added the new hash for the files repo) once I have merged your other pr with the file

Then both things will happen simultaneously.